README.md

```diff
@@ -43,6 +43,7 @@ This will only be available from the machine where the command is executed, but
 
 * The [developer guide](docs/devel/README.md) is for anyone wanting to contribute to Dashboard
 
+* The [troubleshooting guide](docs/user-guide/troubleshooting.md) is a collection of typical setup problems
```

Comment:
I'm worried a bit about discoverability. Ideally this is as close to the installation instructions as possible.

What is the typical doc folks use when they install the dashboard and fail? Is it the [one](http://kubernetes.io/docs/user-guide/ui/) on the Kubernetes docs site? If so, should we somehow link from there? Maybe update those docs to point users to this repo for installation instructions so they can find the troubleshooting doc?

I want the goal of this to be that more folks can successfully install Dashboard without help rather than just less issues for us.